components/Player/PlayerTabs.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import TabBar from '../TabBar';
+import { playerPages } from '../Header/Pages';
+
+const playerPagesMapped = (accountId) => playerPages.map((e) => Object.assign({}, e, {
+  route: `/players/${accountId}/${e.name.toLowerCase()}`,
+  label: e.name,
+}));
+
```

Comment:
could we change this to use the spread operator since it is easier to read and matches the code style we have been using? ex:
```js
const playerPagesMapped = (accountId) => playerPages.map(page => ({
  ...page,
  route: `/players/${accountId}/${e.name.toLowerCase()}`,
  label: e.name,
}));
```